src/minimega/dnsmasq.go

```diff
@@ -70,12 +76,238 @@ to the file.`,
 		},
 		Call: wrapSimpleCLI(cliDnsmasq),
 	},
+	{
+		HelpShort: "configure dhcp/dns options",
+		HelpLong: `
```

Comment:
Please wrap this to 80 characters to keep it consistent with the other help sections. In vim: `gqal` to format the current line.